kill.go

```diff
@@ -102,6 +102,10 @@ func kill(containerID, signal string, all bool) error {
 		return err
 	}
 
+	if signal == "" {
```

Comment:
Here again we do the checking outside of this function already, ensuring signal cannot be ""